tests/Feature/Controllers/AlbumControllerTest.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Tests\Feature\Controllers;
+
+use CountriesSeeder;
+
+use App\Album;
+use App\Contracts\ArtistRepositoryInterface;
+use App\Contracts\AlbumRepositoryInterface;
+use App\Contracts\ProfileRepositoryInterface;
+use App\Contracts\UserRepositoryInterface;
+use App\Http\Requests\StoreAlbum;
+use App\Http\Requests\UpdateAlbum;
+
+class AlbumControllerTest extends ControllerTestCase
+{
+    /**
+     * @var AlbumRepositoryInterface $album
+     */
+    protected $album;
+
+    /**
+     * @var ArtistRepositoryInterface $artist
+     */
+    protected $artist;
+
+    /**
+     * @var ProfileRepositoryInterface $profile
+     */
+    protected $profile;
+
+    /**
+     * @var UserRepositoryInterface $user
+     */
+    protected $user;
+
+    /**
+     * @var StoreAlbum
+     */
+    protected $storeArtist;
+
+    /**
+     * @var UpdateAlbum
+     */
+    protected $updateArtist;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed(CountriesSeeder::class);
+
+        $this->artist = resolve(ArtistRepositoryInterface::class);
+        $this->profile = resolve(ProfileRepositoryInterface::class);
+        $this->user = resolve(UserRepositoryInterface::class);
+
+        $this->album = resolve(AlbumRepositoryInterface::class);
+
+        $this->storeAlbum = new StoreAlbum();
```

Comment:
I copied these over from `ArtistControllerTest`, which I wrote over eight months ago, but never used there. I can't recall what I was thinking, but the way this is done (even if they were actually being used), with the concretion, makes me inclined simply to delete them from both classes. There no value here, correct?

Reply:
I trashed the instances in both classes; I just don't see any usefulness there.